src/snowflake/cli/api/commands/utils.py

```diff
@@ -0,0 +1,20 @@
+from typing import List, Optional
+
+from click import ClickException
+from snowflake.cli.api.commands.structs import Variable
+
+
+def parse_key_value_variables(variables: Optional[List[str]]) -> List[Variable]:
+    """Util for parsing key=value input. Useful for commands accepting multiple input options."""
+    if not variables:
+        return []
+    result: List[Variable] = []
+    if not variables:
+        return result
```

Comment:
i know this is just moved from another module, but we could clean up the duplicated `if not variables` here
```suggestion
    result: List[Variable] = []
    if not variables:
        return result
```

Reply:
Done